zfsbootmenu/lib/zfsbootmenu-core.sh

```diff
@@ -1743,6 +1745,22 @@ load_key() {
   return $?
 }
 
+# arg1: ZFS filesystem
+# prints: The base path to this filesystem for use by ZFSBootMenu functions, with out a trailing /
+# returns: nothing
+
+be_location() {
+  local fs
+  fs="${1}"
+  if [ -z "${fs}" ]; then
+    zerror "fs is undefined"
+    return 1
+  fi
+  zdebug "fs set to ${fs}"
+
+  echo "${BASE}/environments/${fs}"
```

Comment:
Could do
```suggestion
  local beloc="${BASE}/environments/${fs}"
  mkdir -p "${beloc}"
  echo "${beloc}"
```
to make sure the path always exists when a path is returned.

Reply:
Good idea. Done!